app/controllers/api/v1/subscriptions_controller.rb

```diff
@@ -37,7 +37,13 @@ def create
 
       # NOTE: We can't destroy a subscription, it will terminate it
       def terminate
-        subscription = current_organization.subscriptions.active.find_by(external_id: params[:external_id])
+        subscription = current_organization.subscriptions.where(external_id: params[:external_id])
```

Comment:
Maybe since it is an intermediary query variable, you could name it `query`:

```ruby
query = current_organization.subscriptions.where(external_id: params[:external_id])
query = params[:status] == 'pending' ? query.pending : query.active
subscription = query.first
```

Reply:
Done something similar thanks for the proposal love it.

Just har to keep the `if...ifelse` for rubocop to not complain but renames the first variable to query